projects/demo/src/modules/markup/icons/icons.template.html

```diff
@@ -18,7 +18,26 @@
             component and all Taiga UI component accepting icons as input.
         </p>
 
-        <icons-group [icons]="icons[key]"></icons-group>
+        <icons-group [icons]="icons[key]">
+            <ng-template
```

Comment:
What's the purpose of template rather than building this button into the `icons-group` component?

Reply:
for proprietary illustration we need use custom template for it

Reply:
Ok then.